docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
+
+pushd %~dp0
+
+REM Command file for Sphinx documentation
+
+if "%SPHINXBUILD%" == "" (
+	set SPHINXBUILD=sphinx-build
+)
+set SOURCEDIR=source
+set BUILDDIR=build
+
+if "%1" == "" goto help
+
+%SPHINXBUILD% >NUL 2>NUL

```

Comment:
Never seen a `make.bat` how does it work? / When does it get called?

Reply:
As I understood it from the Sphinx documentation, this `.bat` file can be calles from windows users. But I guess we don't need that